.gitignore

```diff
@@ -4,6 +4,7 @@ target
 .cargo
 node_modules
 coverage
+.*.swp
```

Comment:
That should go in a local .gitignore please.

Reply:
Fixed